### PR TITLE
PreparedStatement and CallableStatement move to LocalTransactionContext

### PR DESCRIPTION
**Pull request by HidekiSugimoto189**

### CODE REVIEW

File: src/test/java/jp/co/future/uroborosql/SqlAgentTest.java [hunk 1]
```diff
@@ -392,6 +392,31 @@ public void testQueryMapResuletSetConverter() throws Exception {
 	}
 
 	/**
+	 * クエリ実行処理のテストケース(Fluent API)。
+	 */
+	@Test
+	public void testQueryFluentLamdaAndUpdate() throws Exception {
```
Comment (shout-star) [inline src/test/java/jp/co/future/uroborosql/SqlAgentTest.java]: The method name `testQueryFluentLamdaAndUpdate` is incorrect. You should do as follows.

incollect: `testQueryFluentLamdaAndUpdate`  
collect　:   `testQueryFluentLambdaAndUpdate`

There are other methods that are `Lamda`, so please correct them as well.

File: src/main/java/jp/co/future/uroborosql/fluent/SqlFluent.java [hunk 2]
```diff
@@ -42,6 +43,14 @@
 	T paramList(String paramName, Object... value);
 
 	/**
+	 * 引数として渡されたMapの[key, value]のセットをパラメータに追加<br>
+	 *
+	 * @param paramMap
```
Comment (HidekiSugimoto189) [inline src/main/java/jp/co/future/uroborosql/fluent/SqlFluent.java]: This was my mistake. fixed

File: src/main/java/jp/co/future/uroborosql/context/SqlContextImpl.java [hunk 3]
```diff
@@ -354,7 +354,20 @@ public SqlContext paramList(final String parameterName, final Object... value) {
 	/**
 	 * {@inheritDoc}
 	 *
-	 * @see jp.co.future.uroborosql.context.SqlContext#param(java.lang.String, java.lang.Object, java.sql.SQLType)
+	 * @see jp.co.future.uroborosql.fluent.SqlFluent#paramMap(Map<String, Object>)
```
Comment (HidekiSugimoto189) [inline src/main/java/jp/co/future/uroborosql/context/SqlContextImpl.java]: This was my mistake. fixed

File: src/main/java/jp/co/future/uroborosql/tx/LocalTransactionManager.java [hunk 4]
```diff
@@ -1,31 +1,43 @@
...
 	private final Deque<LocalTransactionContext> txCtxStack = new ConcurrentLinkedDeque<LocalTransactionContext>();
 
-	private Connection unmanagedConnection = null;
+	/** トランザクション管理外の接続に利用する便宜上のトランザクション */
+	private LocalTransactionContext unmanagedTransaction = null;
 
 	/**
 	 * コンストラクタ
 	 *
 	 * @param connectionSupplier コネクションサプライヤ
```
Comment (HidekiSugimoto189) [inline src/main/java/jp/co/future/uroborosql/tx/LocalTransactionManager.java]: This was my mistake. fixed

File: src/test/java/jp/co/future/uroborosql/SqlAgentTest.java (hunk 1 above)
Comment (HidekiSugimoto189) [inline src/test/java/jp/co/future/uroborosql/SqlAgentTest.java]: OK. Changed. 
Other methods same mistake fix it together.

**Comment by HidekiSugimoto189** [comment]
check make javadoc

**Comment by HidekiSugimoto189** [comment]
fixed and push.

**Comment by shout-star** [comment]
Thanks!!

**Pull request merged**